internal/cli/atlas/logs/download.go

```diff
@@ -92,36 +77,36 @@ func (opts *DownloadOpts) newDateRangeOpts() *atlas.DateRangetOptions {
 	}
 }
 
-var logNames = []string{"mongodb.gz", "mongos.gz", "mongodb-audit-log.gz", "mongos-audit-log.gz"}
-
 // mongocli atlas logs download <hostname> <logname> [--type type] [--output destination] [--projectId projectId]
 func DownloadBuilder() *cobra.Command {
 	const argsN = 2
-	opts := &DownloadOpts{
-		fs: afero.NewOsFs(),
-	}
+	opts := &DownloadOpts{}
+	opts.Fs = afero.NewOsFs()
```

Comment:
[q] is there a particular reason for changing this?

Reply:
`FS` comes from the embedded `cli.DownloaderOpts` struct, so I could do, `&DownloadOpts{ DownloaderOpts: DownloaderOpts{ FS: afero.NewOsFs()}}` or make it two lines. I prefer two lines